app/src/main/java/org/p2p/wallet/utils/TabExtensions.kt

```diff
@@ -3,14 +3,31 @@ package org.p2p.wallet.utils
 import android.content.Context
 import android.net.Uri
 import androidx.appcompat.app.AppCompatActivity
+import androidx.browser.customtabs.CustomTabColorSchemeParams
 import androidx.browser.customtabs.CustomTabsIntent
 import androidx.fragment.app.Fragment
+import org.p2p.wallet.R
 
 fun Context.showUrlInCustomTabs(url: String) {
     val uri = Uri.parse(url)
-    val customTabsIntent = CustomTabsIntent.Builder()
-        .setShowTitle(true)
-        .build()
+    val context = this
+    val customTabsIntent = CustomTabsIntent.Builder().apply {
+        setShowTitle(true)
+        val mainBackgroundColor = context.getColor(R.color.backgroundPrimary)
+        val defaultColors = CustomTabColorSchemeParams.Builder()
```

Comment:
i would extract some parts of this `apply` to private methods

so it would be

```
.apply {
  setShowTitle(true)
  setColors(...)
  setDefaultColorSchemeParams(defaultColors)
  setCloseIcon()
```